Rubberduck.UnitTesting/CodeGeneration/TestCodeGeneratorStatics.cs

```diff
@@ -116,12 +116,12 @@ End If
 #End If";
 
         private static string TestModuleTemplate =>
-$@"{{0}}Option Explicit
-Option Private Module
-
-'@TestModule
+$@"{{0}}'@TestModule
```

Comment:
Since marker 0 is for `Option Compare Database` when hosted in Access, it should probably move above `Option Explicit`, to keep tue annotations above all `Option` statements in all situations.

Reply:
I was wondering what the `{{0}}` is good for but couldn't find it out (myself). Thanks for the hint/clarification!